vnext/tsconfig.json

```diff
@@ -1,19 +1,8 @@
 {
+  "extends": "@rnw-scripts/ts-config",
   "compilerOptions": {
     "baseUrl": ".",
-    "target": "es6",
```

Comment:
This was overriden to ES5 earlier, but new target is ES2018, which I think Metro with new Babel should be able to handle fine.